pkg/vcs/vcs_test.go

```diff
@@ -155,3 +157,35 @@ func TestCommitLink(t *testing.T) {
 		}
 	}
 }
+
+func TestParse(t *testing.T) {
+	lines := []string{"Masahiro Yamada <masahiroy@kernel.org> " +
+		"(maintainer:KERNEL BUILD + files below scripts/ (unless mai...)",
+		"Michal Marek <michal.lkml@markovi.net>" +
```

Comment:
Don't split the input lines, it makes it harder to read and easy to miss some whitespaces, or corrupt the input while copy-pasting the script output.
If you got complaints about line length, add this on top of the statement:
```
// nolint: lll
```